commit-regex.js

```diff
@@ -0,0 +1,2 @@
+//This is a common source of truth for both the actual action and unit tests
+module.exports = "^v?(\\d+\\.\\d+\\.\\d+(?:-(?:alpha|beta)\\.\\d+)?)$";
```

Comment:
I think it could make more sense to have a method `matchCommitMessage` or so, exported in index.js, and then call that method in the test. Then there would be no need for this

Reply:
@pascalgn Sorry about that. I've now exported an object containing `commitRegex` to reduce future changes should the need arise to export additional objects.